nodepool/setup_nodepool.yml

```diff
@@ -119,6 +119,15 @@
       include_role:
         name: openstack.logrotate
 
+    - name: Zookeeper TX Log Cleanup Cron
+      cron:
+        name: "zookeeper cleanup"
+        minute: 0
+        hour: 0
+        user: root
+        job: "/usr/share/zookeeper/bin/zkCleanup.sh -n 3 >> /var/log/zookeeper/zookeeper_cron_cleanup.log"
```

Comment:
This log file should be added to the log rotation config.

Reply:
Good catch! Added to log rotation as weekly.